app/gui/src/lib.rs

```diff
@@ -142,23 +142,26 @@ mod profile_workflow;
 #[allow(dead_code)]
 pub fn entry_point_ide() {
     init_tracing(WARN);
-    ensogl_text_msdf_sys::run_once_initialized(|| {
-        // Logging of build information.
-        #[cfg(debug_assertions)]
-        analytics::remote_log_value(
-            "debug_mode",
-            "debug_mode_is_active",
-            analytics::AnonymousData(true),
-        );
-        #[cfg(not(debug_assertions))]
-        analytics::remote_log_value(
-            "debug_mode",
-            "debug_mode_is_active",
-            analytics::AnonymousData(false),
-        );
-
-        let config =
-            crate::config::Startup::from_web_arguments().expect("Failed to read configuration.");
-        crate::ide::Initializer::new(config).start_and_forget();
+    // Logging of build information.
+    #[cfg(debug_assertions)]
+    let debug_mode = true;
+    #[cfg(not(debug_assertions))]
+    let debug_mode = false;
+    analytics::remote_log_value(
+        "debug_mode",
+        "debug_mode_is_active",
+        analytics::AnonymousData(debug_mode),
+    );
+    let config =
```

Comment:
Why is this inlined instead of the previous `start_and_forget` method?

Reply:
`start_and_forget` did not just `start()` and forget; it had logic for hiding the loading indicator, which is not appropriate for any of the other users of `Initializer`.